app/src/main/java/com/example/androidacademyhomework/repository/Repository.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.androidacademyhomework.repository
+
+import android.app.Application
+import com.example.androidacademyhomework.data.loadMovies
+
+class Repository(private val application: Application) {
```

Comment:
Would be good to have proper name for the Repository.

Reply:
fixed